.github/workflows/tests.yml

```diff
@@ -17,6 +17,8 @@ jobs:
 
       - name: Lint
         run: |
+          autoreconf
+          ./configure --enable-coverage
```

Comment:
The check-lint job only performs check on the unbuilt repository, so I do not see the need for running `configure` with `--enable-coverage` here?

Reply:
I couldn't decide either way, but you are right, the `--enable-coverage` is not necessary, so it shouldn't be there.